src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/Constants.java

```diff
@@ -0,0 +1,9 @@
+package uk.nhs.digital.nhsconnect.lab.results.model;
+
+@SuppressWarnings("checkstyle:hideutilityclassconstructor")
```

Comment:
I don't think we should suppress this - just do it. What do you think?
```
private Constants() {}
```

Reply:
Looks like the majority of classes with an empty constructor don't suppress it, so yeah I can add it in